README.md

```diff
@@ -44,18 +44,20 @@ positional arguments:
 options:
   -h, --help            show this help message and exit
   -m MODEL_NAME, --model_name MODEL_NAME
-                        model name to use for generating embeddings, i.e. ada-002, textembedding-gecko, or intfloat/e5-large-v2
+                        model name to use for generating embeddings, i.e. text-embedding-ada-002, textembedding-gecko, or intfloat/e5-large-v2
+  -d DIMENSION_SIZE, --dimension_size DIMENSION_SIZE
```

Comment:
hm maybe we should call this reduced_dimension_size to avoid confusion?

Reply:
Changed to `[-rd REDUCED_DIMENSION_SIZE]`

```
-rd REDUCED_DIMENSION_SIZE, --reduced_dimension_size REDUCED_DIMENSION_SIZE
                        Reduced (output) dimension size. Only supported in models (e.g. OpenAI text-embedding-3-xxx) that have this feature. Ignored otherwise!
```